Add `except: ["exact-value"]` to `media-feature-range-notation` (#8675)
stylelint/stylelint: lib/rules/media-feature-range-notation/README.md

```css
@media (min-width: 1px) and (max-width: 2px) {}
```

## Optional secondary options

### `except`

```json
{ "except": ["array", "of", "options"] }
```

#### `"exact-value"`

> Ignore media features with exact values when the `context` option is selected.

Reviewer: ```suggestion
Reverse the primary option for media features with exact values.
```

To be consistent with our [`except` options](https://stylelint.io/developer-guide/rules#add-options) in other rules, let's have it reverse the primary option.

Let's add tests for:

```json
{
  "media-feature-range-notation": ["prefix", { "except": ["exact-value"] }]
}
```

which allows:

```css
@media (width = 1px) {}
```

```css
@media (min-width: 1px) {}
```

and disallows:

```css
@media (width > 1px) {}
```

```css
@media (width: 1px) {}
```

No need to add docs for this, as we typically only show one example of the primary option being reversed.

Given:

```json
{
  "media-feature-range-notation": ["context", { "except": ["exact-value"] }]
}
```

> 

Reviewer: Let's add an example of:

```css
@media (min-width: 1px) {}
```

As a pattern that'll be a problem, as we typically include both for `except` options.

Author: Added
